2025-10-21 18:31:07,429 [WARNING] PyNaCl is not installed, voice will NOT be supported
2025-10-21 18:31:12,091 [INFO] logging in using static token
2025-10-21 18:31:12,091 [INFO] Session ID: cd5205d2876caa05cae25b52bd655931c63bb38fe5ac32ffd6559b6104028a16
2025-10-21 18:31:13,536 [INFO] Shard ID None has connected to Gateway (Session ID: bcaba5c24378019413dbb6e2a841f73f).
2025-10-21 18:31:15,549 [INFO] Logged in as BestBotEver!!!#9403 (ID: 1386280922218893465)
2025-10-21 18:31:15,549 [INFO] No startmessage set.
2025-10-21 18:31:17,963 [INFO] 1430156500441104434:p.pumpum (1260198579067420722) in #ü§ñ-bot-test-üõ†Ô∏è (1399900695993253970): !jvc
2025-10-21 18:31:17,965 [INFO] User p.pumpum has GOD role, skipping banned word check.
2025-10-21 18:31:17,966 [INFO] [COMMAND] jvc called by p.pumpum | mode=None | id=None
2025-10-21 18:31:17,966 [INFO] [DEBUG] Detected caller's channel: Room//GamingüéÆ (1418976416220971079)
2025-10-21 18:31:17,967 [INFO] [INFO] Attempt 1: connecting to Room//GamingüéÆ (1418976416220971079)...
2025-10-21 18:31:17,967 [ERROR] [ERROR] Unexpected connection error: PyNaCl library needed in order to use voice
Traceback (most recent call last):
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\main.py", line 1478, in attempt_connect
    vc = await channel.connect(reconnect=False)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\lib\discord\abc.py", line 1984, in connect
    voice: T = cls(client, self)
               ^^^^^^^^^^^^^^^^^
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\lib\discord\voice_client.py", line 220, in __init__
    raise RuntimeError("PyNaCl library needed in order to use voice")
RuntimeError: PyNaCl library needed in order to use voice
2025-10-21 18:31:22,991 [INFO] [INFO] Attempt 2: connecting to Room//GamingüéÆ (1418976416220971079)...
2025-10-21 18:31:22,991 [ERROR] [ERROR] Unexpected connection error: PyNaCl library needed in order to use voice
Traceback (most recent call last):
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\main.py", line 1478, in attempt_connect
    vc = await channel.connect(reconnect=False)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\lib\discord\abc.py", line 1984, in connect
    voice: T = cls(client, self)
               ^^^^^^^^^^^^^^^^^
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\lib\discord\voice_client.py", line 220, in __init__
    raise RuntimeError("PyNaCl library needed in order to use voice")
RuntimeError: PyNaCl library needed in order to use voice
2025-10-21 18:31:27,999 [INFO] [INFO] Attempt 3: connecting to Room//GamingüéÆ (1418976416220971079)...
2025-10-21 18:31:27,999 [ERROR] [ERROR] Unexpected connection error: PyNaCl library needed in order to use voice
Traceback (most recent call last):
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\main.py", line 1478, in attempt_connect
    vc = await channel.connect(reconnect=False)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\lib\discord\abc.py", line 1984, in connect
    voice: T = cls(client, self)
               ^^^^^^^^^^^^^^^^^
  File "d:\TonpalmUnmain\Project\BestDiscordBotEver\lib\discord\voice_client.py", line 220, in __init__
    raise RuntimeError("PyNaCl library needed in order to use voice")
RuntimeError: PyNaCl library needed in order to use voice
2025-10-21 18:31:33,480 [INFO] 1430156565448626227:BestBotEver!!!#9403 (1386280922218893465) in #ü§ñ-bot-test-üõ†Ô∏è (1399900695993253970): ‚ùå Failed to connect after multiple attempts.
2025-10-21 18:31:33,582 [ERROR] [FATAL] Failed to connect to Room//GamingüéÆ after retries.
2025-10-21 18:31:33,583 [INFO] Command completed: jvc by p.pumpum
